secrets.sample.js

```diff
@@ -0,0 +1,2 @@
+// You can register your app and get a client ID at https://apps.dev.microsoft.com
+window.ClientId = "<insert id here>";
```

Comment:
While not part of this checkin, what happens if they don't add the client id here? Hopefully they get a message when they run the explorer that tells them to add the clientId.